{"repository":"https://github.com/z-t-y/flog","dockerfilePath":"Dockerfile","startTime":1673032521117,"endTime":1673032938477,"clone":{"startTime":1673032521117,"stdout":"","stderr":"Cloning into '/tmp/dinghy-analysis/z-t-y/flog'...\n","error":null,"endTime":1673032522186,"commit":"0f3621e63406150e490eec0d40ad21a59e257c5c"},"originalBuild":{"startTime":1673032522261,"endTime":1673032938396,"error":{"code":1,"killed":false,"signal":null,"cmd":"docker build --no-cache --force-rm -t flog:latest -f /tmp/dinghy-analysis/z-t-y/flog/Dockerfile ."},"stdout":"Sending build context to Docker daemon  1.768MB\r\r\nStep 1/15 : FROM python:3.9-slim\n3.9-slim: Pulling from library/python\n3f4ca61aafcd: Already exists\n3f487a3359db: Already exists\nae22731824be: Pulling fs layer\n3583ac268677: Pulling fs layer\nde224c04316a: Pulling fs layer\n3583ac268677: Verifying Checksum\n3583ac268677: Download complete\nde224c04316a: Verifying Checksum\nde224c04316a: Download complete\nae22731824be: Verifying Checksum\nae22731824be: Download complete\nae22731824be: Pull complete\n3583ac268677: Pull complete\nde224c04316a: Pull complete\nDigest: sha256:9e0b4391fc41bc35c16caef4740736b6b349f6626fd14eba32793ae3c7b01908\nStatus: Downloaded newer image for python:3.9-slim\n ---> e2f464551004\nStep 2/15 : WORKDIR /usr/src/flog/\n ---> Running in 87711de94230\nRemoving intermediate container 87711de94230\n ---> 1f888f687f37\nStep 3/15 : COPY Pipfile* ./\n ---> 0f19ed3d3e0e\nStep 4/15 : RUN pip install pipenv -i https://mirrors.aliyun.com/pypi/simple --trusted-host mirrors.aliyun.com\n ---> Running in 0924ed0e4d11\nLooking in indexes: https://mirrors.aliyun.com/pypi/simple\nCollecting pipenv\n  Downloading https://mirrors.aliyun.com/pypi/packages/d3/94/49d389bf66be0aea4a82adb0f00f6057e66934878694350712bb284ec099/pipenv-2022.12.19-py3-none-any.whl (2.9 MB)\n     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.9/2.9 MB 35.2 kB/s eta 0:00:00\nCollecting virtualenv-clone>=0.2.5\n  Downloading https://mirrors.aliyun.com/pypi/packages/21/ac/e07058dc5a6c1b97f751d24f20d4b0ec14d735d77f4a1f78c471d6d13a43/virtualenv_clone-0.5.7-py3-none-any.whl (6.6 kB)\nCollecting certifi\n  Downloading https://mirrors.aliyun.com/pypi/packages/71/4c/3db2b8021bd6f2f0ceb0e088d6b2d49147671f25832fb17970e9b583d742/certifi-2022.12.7-py3-none-any.whl (155 kB)\n     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 155.3/155.3 KB 35.2 kB/s eta 0:00:00\nRequirement already satisfied: setuptools>=36.2.1 in /usr/local/lib/python3.9/site-packages (from pipenv) (58.1.0)\nCollecting virtualenv\n  Downloading https://mirrors.aliyun.com/pypi/packages/18/a2/7931d40ecb02b5236a34ac53770f2f6931e3082b7a7dafe915d892d749d6/virtualenv-20.17.1-py3-none-any.whl (8.8 MB)\n     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 8.8/8.8 MB 32.6 kB/s eta 0:00:00\nCollecting distlib<1,>=0.3.6\n  Downloading https://mirrors.aliyun.com/pypi/packages/76/cb/6bbd2b10170ed991cf64e8c8b85e01f2fb38f95d1bc77617569e0b0b26ac/distlib-0.3.6-py2.py3-none-any.whl (468 kB)\n     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.5/468.5 KB 35.0 kB/s eta 0:00:00\nCollecting filelock<4,>=3.4.1\n  Downloading https://mirrors.aliyun.com/pypi/packages/14/4c/b201d0292ca4e0950f0741212935eac9996f69cd66b92a3587e594999163/filelock-3.9.0-py3-none-any.whl (9.7 kB)\nCollecting platformdirs<3,>=2.4\n  Downloading https://mirrors.aliyun.com/pypi/packages/c1/c7/9be9d651b93efce682b45142a6267034fc4215972780748618c02e236361/platformdirs-2.6.2-py3-none-any.whl (14 kB)\nInstalling collected packages: distlib, virtualenv-clone, platformdirs, filelock, certifi, virtualenv, pipenv\nSuccessfully installed certifi-2022.12.7 distlib-0.3.6 filelock-3.9.0 pipenv-2022.12.19 platformdirs-2.6.2 virtualenv-20.17.1 virtualenv-clone-0.5.7\n\u001b[91mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\n\u001b[0m\u001b[91mWARNING: You are using pip version 22.0.4; however, version 22.3.1 is available.\nYou should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.\n\u001b[0mRemoving intermediate container 0924ed0e4d11\n ---> e450d2cf1a9b\nStep 5/15 : RUN pipenv install --dev --system --deploy\n ---> Running in fdf66941703d\n\u001b[91mTraceback (most recent call last):\n  File \"/usr/local/bin/pipenv\", line 8, in <module>\n    sys.exit(cli())\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/vendor/click/core.py\", line 1128, in __call__\n\u001b[0m\u001b[91m    return self.main(*args, **kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/cli/options.py\", line 57, in main\n\u001b[0m\u001b[91m    return super().main(*args, **kwargs, windows_expand_args=False)\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/vendor/click/core.py\", line 1053, in main\n\u001b[0m\u001b[91m    rv = self.invoke(ctx)\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/vendor/click/core.py\", line 1659, in invoke\n\u001b[0m\u001b[91m    return _process_result(sub_ctx.command.invoke(sub_ctx))\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/vendor/click/core.py\", line 1395, in invoke\n\u001b[0m\u001b[91m    return ctx.invoke(self.callback, **ctx.params)\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/vendor/click/core.py\", line 754, in invoke\n\u001b[0m\u001b[91m    return __callback(*args, **kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/vendor/click/decorators.py\", line 84, in new_func\n\u001b[0m\u001b[91m    return ctx.invoke(f, obj, *args, **kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/vendor/click/core.py\", line 754, in invoke\n\u001b[0m\u001b[91m    return __callback(*args, **kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/cli/command.py\", line 239, in install\n\u001b[0m\u001b[91m    do_install(\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/core.py\", line 2273, in do_install\n\u001b[0m\u001b[91m    do_init(\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/core.py\", line 1339, in do_init\n\u001b[0m\u001b[91m    new_hash = project.calculate_pipfile_hash()\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/project.py\", line 1053, in calculate_pipfile_hash\n\u001b[0m\u001b[91m    p = plette.Pipfile.load(pf)\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/vendor/plette/pipfiles.py\", line 65, in load\n\u001b[0m\u001b[91m    return cls(data)\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/vendor/plette/models/base.py\", line 49, in __init__\n\u001b[0m\u001b[91m    self.validate(data)\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/vendor/plette/pipfiles.py\", line 44, in validate\n\u001b[0m\u001b[91m    klass.validate(data[key])\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/vendor/plette/models/base.py\", line 149, in validate\n\u001b[0m\u001b[91m    cls.item_class.validate(d)\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/vendor/plette/models/base.py\", line 81, in validate\n\u001b[0m\u001b[91m    return validate(cls, data)\n  File \"/usr/local/lib/python3.9/site-packages/pipenv/vendor/plette/models/base.py\", line 38, in validate\n\u001b[0m\u001b[91m    raise ValidationError(data, v)\n\u001b[0m\u001b[91mpipenv.vendor.plette.models.base.ValidationError: {'name': 'pypi', 'url': 'https://mirrors.aliyun.com/pypi/simple/', 'verify-ssl': True}\nverify_ssl: required field\n\u001b[0mRemoving intermediate container fdf66941703d\n","stderr":"The command '/bin/sh -c pipenv install --dev --system --deploy' returned a non-zero code: 1\n"}}